src/BaGet.UI/src/DisplayPackage/DisplayPackage.tsx

```diff
@@ -247,6 +248,10 @@ class DisplayPackage extends React.Component<IDisplayPackageProps, IDisplayPacka
               <Dependents packageId={this.state.package.id} />
             </ExpandableSection>
 
+            <ExpandableSection title="Release Notes" expanded={false}>
```

Comment:
Most packages don't have release notes. Let's hide the section if the package is missing release notes:

```ts
{this.state.package.releaseNotes &&
  <ExpandableSection title="Release Notes" expanded={false}>
    <div className="package-release-notes" >{this.state.package.releaseNotes}</div>
  </ExpandableSection>
}
```